app/src/main/java/app/accrescent/client/ui/AppList.kt

```diff
@@ -39,10 +39,17 @@ fun AppList(
     viewModel: AppListViewModel = viewModel(),
     filter: (installStatus: InstallStatus) -> Boolean = { true },
     noFilterResultsText: String = "",
+    searchQuery: String
 ) {
     val apps by viewModel.apps.collectAsState(emptyList())
     val installStatuses = viewModel.installStatuses
     val filteredApps = apps.filter { filter(installStatuses[it.id] ?: InstallStatus.LOADING) }
+        .filter {
+            it.name.lowercase().contains(searchQuery.lowercase())
+        }
+        .sortedBy {
+            it.name.lowercase().indexOf(searchQuery.lowercase())
+        }
```

Comment:
nit: Each of these can fit on a single line rather than three, i.e.,

```kotlin
.filter { it.name.lowercase().contains(searchQuery.lowercase()) }
.sortedBy { it.name.lowercase().indexOf(searchQuery.lowercase()) }
```

Generally this style is preferred throughout the codebase unless it significantly hinders readability. I could probably make that more clear in the contributing guidelines.